shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -135,6 +143,21 @@ FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopViewCreateFromNewWindow(
     const FlutterDesktopWindowProperties& window_properties,
     FlutterDesktopEngineRef engine);
 
+// Creates a view that hosts and displays the given engine instance.
+FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopViewCreateFromElmParent(
+    const FlutterDesktopViewProperties& view_properties,
+    FlutterDesktopEngineRef engine,
+    void* parent);
+
+// Returns a handle to evas object that the FlutterView is drawn to.
+FLUTTER_EXPORT void* FlutterDesktopViewGetEvasObject(
+    FlutterDesktopViewRef view);
+
+// Resize the FlutterView.
+FLUTTER_EXPORT void FlutterDesktopViewResizeView(FlutterDesktopViewRef view,
+                                                 int32_t width,
+                                                 int32_t height);
```

Comment:
These are still under development. Could you please add appropriate comments to prevent these APIs from being used in commercial applications?

Reply:
I added description.Please review again if desc is appropriate